sdk/prelude/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "miden-sdk-types"
-description = "Miden SDK types"
+name = "miden-prelude"
```

Comment:
I think we probably need a better name for this crate, as `miden-prelude` is super non-descript. More generally we should probably make sure this (and other crates which are part of the Rust bindings to Miden SDK code written in Miden Assembly) uses some kind of convention. Something like `miden-bindgen-<category>`, where `<category>` describes what bindings are provided by the crate.

We aren't currently generating the bindings, so `bindgen` might seem a bit misleading, but we may be able to do so in the future, and for now it conveys a decent idea of what the crate is about. We could probably go even more specific, but I'm not sure how verbose we really want to be.

Not a huge priority though, just something I wanted to note.

Reply:
Good point! I was coming to this name from the user perspective. I want them to import only one crate whether they target Miden rollup or pure Miden VM. The rollup users would use the `miden-sdk` crate that re-exports `miden-prelude` crate and `miden-sdk-tx-kernel` crate. For the users that target pure Miden VM (without rollup) I want to provide intrinsics and stdlib in one crate and I initially wanted to use the name `miden-stdlib`, but it's already taken by one of the VM crates (see https://crates.io/crates/miden-stdlib) so the closest name to intrinsics+stdlib I could think of was `miden-prelude`. 

If I understand your idea, we will end up with the following crates:
`miden-bindgen-intrinsics` - for the bindings to the intrinsics functions
`miden-bindgen-stdlib` - for the bindings to the stdlib functions
`miden-bindgen-tx-kernel` - for the bindings to the tx_kernel functions

We can re-export all of them in the `miden-sdk` crate for the rollup users and in `miden-?`(what name if not `prelude`) crate for the pure Miden VM users. What do you think?